Assets/Tests/InputSystem/CoreTests_Actions.cs

```diff
@@ -1443,6 +1443,40 @@ public void Actions_CanDisableAndEnableOtherAction_FromCallback()
         Assert.That(receivedCalls, Is.EqualTo(2));
     }
 
+    [Test]
+    [Category("Actions")]
+    public void Actions_CanDisableAndEnable_FromCallbackWhileOtherCompositeBindingIsProgress()
+    {
+        // Enables "Modifier must be pressed first" behavior on all Composite Bindings
+        InputSystem.settings.shortcutKeysConsumeInput = true;
+
+        var keyboard = InputSystem.AddDevice<Keyboard>();
+        var map = new InputActionMap("map");
+
+        var withModiferReceivedCalls = 0;
+        var actionWithModifier = map.AddAction("OneModifier", type: InputActionType.Button);
+        actionWithModifier.AddCompositeBinding("OneModifier")
+            .With("Binding", "<Keyboard>/space")
+            .With("Modifier", "<Keyboard>/ctrl");
+        actionWithModifier.performed += _ => ++ withModiferReceivedCalls;
+
+        var actionWithoutModifier = map.AddAction("One", type: InputActionType.Button, binding: "<Keyboard>/space");
+        actionWithoutModifier.performed += _ => actionWithModifier.Disable();
+
+        map.Enable();
+
+        PressAndRelease(keyboard.spaceKey);
```

Comment:
Minor: Could be helpful to describe with a one liner what the stimuli and assertion here is expected to do to make it easier to come back to this test later on.

Reply:
I added more comments